_code-samples/get-started/js/base.js

```diff
@@ -1,17 +1,11 @@
 // In browsers, use a <script> tag. In Node.js, uncomment the following line:
```

Comment:
Since the following line is already uncommented, I think we need to adjust this.

```suggestion
// You can also use a <script> tag in browsers or require('xrpl') in Node.js
```